pipeline/src/test_util.rs

```diff
@@ -516,3 +518,19 @@ pub fn run_reparse_test_with_blacklist(file: &str, blacklist: &[&str]) {
         .compute_optimized_pil()
         .unwrap();
 }
+
+// TODO: Add #[cfg(feature = "stwo")] to conditionally compile this function. now when I add it, this code is disabled. how to eable by default?
+pub fn test_stwo(file_name: &str, inputs: Vec<Mersenne31Field>) {
+    let backend = powdr_backend::BackendType::Stwo;
+    let mut pipeline = Pipeline::default()
+        .with_tmp_output()
+        .from_file(resolve_test_file(file_name))
+        .with_prover_inputs(inputs.clone())
+        .with_backend(backend, None)
+        .set_witness(vec![("add::a".to_string(),
+        vec![Mersenne31Field::from(1),Mersenne31Field::from(2),Mersenne31Field::from(1),Mersenne31Field::from(2)]),("add::b".to_string(), 
+        vec![Mersenne31Field::from(1),Mersenne31Field::from(2),Mersenne31Field::from(1),Mersenne31Field::from(2)]),("add::c".to_string(), 
+        vec![Mersenne31Field::from(2),Mersenne31Field::from(4),Mersenne31Field::from(2),Mersenne31Field::from(4)])]);
```

Comment:
Do we have to build the witness ourselves here? `Pipeline` has a `compute_witness` method which does it automatically for any program, as opposed to the custom approach here.